tsl/test/expected/bgw_db_scheduler.out

```diff
@@ -560,9 +560,10 @@ SELECT * FROM sorted_bgw_log;
       0 |    800000 | DB Scheduler     | [TESTING] Wait until 984375, started at 800000
       1 |    984375 | DB Scheduler     | [TESTING] Registered new background worker
       2 |    984375 | DB Scheduler     | [TESTING] Wait until 1325000, started at 984375
-      1 |    984375 | test_job_2       | job 1001 threw an error
-      2 |    984375 | test_job_2       | Error job 2
-(24 rows)
+      1 |    984375 | test_job_2       | job 1001 reached max_retries after 5 consecutive failures
+      2 |    984375 | test_job_2       | job 1001 threw an error
+      3 |    984375 | test_job_2       | Error job 2
+(25 rows)
```

Comment:
Could you add  a test that shows that we can alter the job config (after max_retries is reached) and run the job again?  (please also add a comment about the use case for the test in the .sql file when you add the test). Is it possible with mock bgw framework?

Reply:
Added some more tests to address it.